help/api/tou.md

```diff
@@ -0,0 +1,121 @@
+# Terms of Use for arXiv APIs
+
+Our mission is to provide rapid dissemination of scientific results at no cost
+to authors or readers. Providing free Application Programming Interfaces (APIs)
+helps us to advance that mission by enabling platforms and projects that extend
+the discoverability of arXiv e-prints and provide valuable services to
+scientists and interested readers. Our APIs include our [OAI-PMH
+interface](../oa), [RSS feeds](../rss), the [legacy arXiv API](index), [bulk
+data downloads](../bulk_data_s3), [SWORD bulk deposit API](../submit_sword),
+and services available via the [arXiv API Gateway](https://api.arxiv.org/).
+Understanding and observing the following terms of use will help us to ensure
+that arXiv continues to be able to fulfill its mission. For more information,
+see arXiv’s [Operating
+Principles](https://confluence.cornell.edu/display/arxivpub/arXiv+Public+Wiki?preview=/340902451/340902455/arXivPrinciplesMarch12.pdf).
+
+By using arXiv APIs, you are agreeing to the terms and conditions outlined on
+this page.
+
+- You understand that e-prints from arXiv are subject to all applicable
+  copyright protections. The redistribution of e-prints requires permission
+  from the copyright holder, which may be the author(s) or publisher. arXiv is
+  not the copyright holder on any of the e-prints available through the API. In
+  some cases, submitters have provided permission in advance by submitting
+  their e-print under a permissive Creative Commons license. The vast majority
+  of e-prints are submitted under the [arXiv.org non-exclusive right to
+  distribute](https://arxiv.org/licenses/nonexclusive-distrib/1.0/license.html).
+  For more information about licenses, see https://arxiv.org/help/license.
+- You are free to use descriptive [metadata](https://arxiv.org/help/prep)<sup>1</sup> about arXiv e-prints for
+  non-commercial purposes, under the terms of the [Creative Commons Universal
```

Comment:
I thought all metadata could be used for commercial and non-commercial purposes. Is this a change?

Reply:
Looks like I overlooked changing the first clause after we added the CC0 language. Will correct.

Reply:
Fixed in 2bc5edd